cmd/minikube/cmd/image.go

```diff
@@ -396,6 +397,7 @@ func init() {
 	saveImageCmd.Flags().BoolVar(&imgDaemon, "daemon", false, "Cache image to docker daemon")
 	saveImageCmd.Flags().BoolVar(&imgRemote, "remote", false, "Cache image to remote registry")
 	imageCmd.AddCommand(saveImageCmd)
+	listImageCmd.Flags().StringVarP(&format, "format", "", "short", "Format output. One of: short|table|json|yaml")
```

Comment:
```suggestion
	listImageCmd.Flags().StringVar(&format, "format", "short", "Format output. One of: short|table|json|yaml")
```
The `P` is for adding a shorthand, but since you're not using the shorthand can omit it.

Reply:
Fixed